pennylane/resource/specs.py

```diff
@@ -43,9 +69,25 @@ def specs(qnode, max_expansion=None, expansion_strategy=None):
             - ``device``: The QNode will attempt to decompose the internal circuit
               such that all circuit operations are natively supported by the device.
 
+        max_expansion (int): The number of times the internal circuit should be expanded when
+            calculating the specification. Defaults to ``qnode.max_expansion``.
+
     Returns:
         A function that has the same argument signature as ``qnode``. This function
-        returns a dictionary of information about qnode structure.
+        returns a dictionary (or a list of dictionaries) of information about qnode structure.
+
+    .. note::
+
+        At most, one of ``level`` or ``expansion_strategy`` needs to be provided. If neither is provided,
+        ``qnode.expansion_strategy`` would be used instead. Users are encouraged to predominantly use ``level``,
+        as it allows for the same values as ``expansion_strategy``, and allows for more flexibility choosing
+        the wanted transforms/expansions.
```

Comment:
We can deprecate `expansion_strategy` in the next release cycle :+1:

Reply:
One of the main reasons I'm currently pushing on a `qml.decompose` or `qml.expand` to get added.

Reply:
Why would adding `qml.decompose` or `expand` help with deprecating `expansion_strategy` here?

Reply:
By placing it in the transform program, and then stepping through the transform program, users can calculate draw and specs at different stages in the decomposition.